api/column_family.cc

```diff
@@ -135,9 +135,9 @@ static future<json::json_return_type>  get_cf_histogram(http_context& ctx, const
 static future<json::json_return_type> get_cf_histogram(http_context& ctx, utils::timed_rate_moving_average_summary_and_histogram replica::column_family_stats::*f) {
     std::function<utils::ihistogram(const replica::database&)> fun = [f] (const replica::database& db)  {
         utils::ihistogram res;
-        for (auto i : db.get_tables_metadata()._column_families) {
-            res += (i.second->get_stats().*f).hist;
-        }
+        db.get_tables_metadata().for_each_table([&] (table_id, lw_shared_ptr<replica::column_family> cf) mutable {
```

Comment:
class table is synonymous with class column_family.
We should deprecate the latter and use `table` in new code.